### PR TITLE
Bumps the FluxHelmRelease API version to v1alpha2

### LINKED ISSUE #1059: Update the FluxHelmRelease CRD API version
**Problem**

1) The current FluxHelmRelease API version is the non standard v1alpha.
2) The Spec.Values format has changed, so we should bump up the version.

3) Current deploy helm-operator manifest refers to the helm-operator image tagged as alpha. This image is currently incompatible with the existing FluxHelmRelease definition.

**A/C**

a) helm-operator working after FluxHelmRelease API version change to  v1alpha2
b) helm-operator manifest for standalone deployment refers to 0.1.0-alpha

### PR DESCRIPTION
**Pull request by tamarakaufler**
Reasons:
- Incorrect initial API version, inconsistent with the Kubernetes one
- Breaking change in the FluxHelmRelease schema
(https://github.com/weaveworks/flux/pull/1035)

Note:
Associated changes made also in the example user repo
weaveworks/flux-helm-test (https://github.com/weaveworks/flux-helm-test/pull/6)

Fixes #1059

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -34,24 +34,7 @@
 [[projects]]
   branch = "master"
   name = "github.com/docker/distribution"
-  packages = [
```
Comment (aaron7) [inline Gopkg.lock]: Please upgrade your local `dep` to the latest version :)

Comment (tamarakaufler) [reply Gopkg.lock]: Thanks for reminding. 

I have done (go get -u github.com/golang/dep/cmd/dep) but no change after doing dep ensure!. Will look into it.

Comment (tamarakaufler) [reply Gopkg.lock]: Running dep version gives:

dep:
 version     : devel
 build date  : 
 git hash    : 
 go version  : go1.8.3
 go compiler : gc
 platform    : linux/amd64
 features    : ImportDuringSolve=false

Comment (tamarakaufler) [reply Gopkg.lock]: Looking at git commits in the `src/github.com/golang/dep/cmd/dep` dir (master branch) shows:

```commit f9cb443b21244c2c2b8f9281014936300793eea6
Merge: 342e0b7 f217a9c
Author: Carolyn Van Slyck <me@carolynvanslyck.com>
Date:   Tue Apr 24 09:20:03 2018 -0500

    Merge pull request #1779 from wknapik/patch-1
    
    Updated Arch installation instructions.
```

Comment (tamarakaufler) [reply Gopkg.lock]: All sorted in the end though lots of magic going on. Thanks for helping out Aaron.

**Pull request merged**